content/compute-at-sdsc.md

```diff
@@ -0,0 +1,68 @@
+Title: Bring your computing to the San Diego Supercomputer Center
+Date: 2018-10-24 18:00
+Author: Andrea Zonca
+Tags: comet, sdsc, xsede
+Slug: compute-at-sdsc
+
+I am often asked what computing resources are available at the San Diego Supercomputer Center for scientists and what is the best way to be granted access. I decided to write a blog post with an overview of all the options, consider that I'm writing this in October 2018, so please cross-check on the official websites.
+
+## Comet
+
+Our key resource is the Comet Supercomputer, a 2000 nodes traditional supercomputer with 72 GPU nodes, each with 4 GPUs.
+Comet has powerful CPUs with 24 cores and lots of memory per node (128GB) and a very fast local flash drive on each node.
+It is also suitable to run large amounts of single node jobs, so you can exploit it even if you don't have a multi-node parallel software.
+
+Comet is a XSEDE resource, XSEDE is basically a consortium of many large US supercomputers dedicated to Science,  it reviews applications from US scientists and grants them supercomputing resources for free. It is funded by National Science Foundation.
+
+### How to request resources on Comet
+
+Ordered from the easiest / lower amount of resources to the largest allocations.
+
```

Comment:
Which means they are ordered by the amount of effort it takes to get each type of allocation.